src/universal/utils/validateSecretToken.js

```diff
@@ -0,0 +1,31 @@
+/* A secret token is a reset or email verification token, not a JWT*/
+export default function validateSecretToken(secretToken) {
```

Comment:
oh yeah... with auth0 & the current scope we probably won't need to make secret tokens, but if we need it, it's a really cheap way to avoid an extra server call. we can always remove this & add it back in later. i may have a few other meatier remnants around